JBPM-8391: Combined view of task comments and task work tabs (#1587)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.22.0.Final/combined_view_task_comments_work_tab-chapter.adoc

[[_jbpmreleasenotes72201]]

> == Combined view of task comments and task work tab

Reviewer: I would probably highlight the main advantage: View task comments in Work tab, something like that.

Author: oks

> A new configuration parameter has been added to display the task comments directly at task *Work tab*. If the *Show

Reviewer: Starting from this release, users will be able to visualize the most recent Task related comments alog with the Task form at the Work tab. 

Then add a screenshot here?

A new configuration parameter has been....

> A new configuration parameter have been added to be able to display the task comments at task *Work tab*. if the *Show

Reviewer 2: ```suggestion
A new configuration parameter has been added to display the task comments directly at task *Work tab*. If the *Show
```

>  task comments at work tabs* is set then,  under the task form, the related comments will be displayed.

Reviewer 2: ```suggestion
 task comments at work tabs* is set then, under the task form, the related comments will be displayed.
```

As before if the task is not Completed, new comments can be added there too. The design of the comments list has been

>  changed too.

Reviewer: has been enhanced.

> As before if the task is not Completed, new comments can be added there too. The design of the comments list have been

Reviewer 2: ```suggestion
As before if the task is not Completed, new comments can be added there too. The design of the comments list has been
```

The list of displayed comments at *Work Tab* only shows the first three comments, and a *load more comments*
 button would appear if there are more comments to load.

> At task *Comments Tab* tab the same comments list design is displayed but there, the list shows the first

Reviewer 2: ```suggestion
At task *Comments Tab* tab the list of comments follows the same design, except that the list shows the first
```

 ten comments, keeping the same behaviour with the load button if there are more comments to load.

> Both list allow sorting the comments by the comment creation date(ASC /DESC). Notice that the user only can delete

Reviewer 2: ```suggestion
Both lists allow sorting the comments by the comment creation date(ASC/DESC). Notice that the users only can delete
```

>  comments added by itself having the remove action available only on its own comments.

Reviewer 2: ```suggestion
 comments added by themselves having the remove action available only on their own comments.
```

[[_jbpmreleasenotes72201]]

> == View task comments at Work tab

Reviewer 3: @nmirasch, I suggest the following simplified rewrite. Feel free to adopt or adapt as you see fit. Imo, the changed behavior and benefit is what matters, and trivial details that are intuitive in the UI itself are unnecessary.

```
== View task comments in Work tab
You can now add and view task-related comments in the *Work* tab of tasks in {CENTRAL}. This enhancement creates more streamlined workflow and collaboration around tasks.

image::ReleaseNotes/TaskCommentsAtWorkTab_7.22.0.png[align="center", title="Task Work tab displaying comments"]

To enable or disable this feature, you can select or clear the *Show
 task comments at work tab* parameter in the process administration settings in {CENTRAL}.

image::ReleaseNotes/TaskCommentsAtWorkTabConf_7.22.0.png[align="center", title="Process Administration settings"]

You cannot add comments to tasks in a *Completed* state and you cannot delete comments added by other users.
```

Author: @sterobin updated, simpler and clearer, Thank you very much!
